hickle/loaders/load_astropy.py

```diff
@@ -219,23 +217,19 @@ def check_is_astropy_quantity_array(py_obj):
         return False
 
 
-#####################
-# Lookup dictionary #
-#####################
-
+# %% REGISTERS
 class_register = [
```

Comment:
Random comment: even as I implemented this `class_register` approach I thought it was pretty gross. I suspect after editing it you agree :stuck_out_tongue_closed_eyes: ! Not sure what the straightforward improvement is though.